pkg/action/scan.go

```diff
@@ -58,6 +58,8 @@ func findFilesRecursively(ctx context.Context, root string, c Config) ([]string,
 
 // cleanPath removes the temporary directory prefix from the path.
 func cleanPath(path string, prefix string) string {
+	// remove any "/private" prefixes from the path
+	path = strings.TrimPrefix(path, "/private")
```

Comment:
This seems like a fragile workaround. Would using [EvalSymlinks](https://cs.opensource.google/go/go/+/go1.22.4:src/path/filepath/path.go;l=285) [¶](https://pkg.go.dev/path/filepath#EvalSymlinks) give you the result you are looking for?

Reply:
Indeed!
```
// path after eval
/private/var/folders/2v/jvdj3pbd31g8q_m_70d094zr0000gn/T/apko_nested.tar.gz2845949861/apko_0.13.2_linux_arm64/apko 

// prefix after eval
/private/var/folders/2v/jvdj3pbd31g8q_m_70d094zr0000gn/T/apko_nested.tar.gz2845949861
```